platform/linux-dpdk/odp_crypto.c

```diff
@@ -784,8 +832,9 @@ int odp_crypto_session_create(odp_crypto_session_param_t *param,
 	uint8_t cdev_id = 0;
 	struct rte_crypto_sym_xform cipher_xform;
 	struct rte_crypto_sym_xform auth_xform;
-	struct rte_crypto_sym_xform *first_xform;
+	struct rte_crypto_sym_xform *first_xform = NULL;
 	struct rte_cryptodev_sym_session *session;
+	enum crypto_chain_order res = CRYPTO_CHAIN_NOT_SUPPORTED;
```

Comment:
`res` does not quite follow `crypto_chain_order`, does it? Please rename to 'order' or smth. like that.